Source/Core/UpdaterCommon/UpdaterCommon.cpp

```diff
@@ -319,8 +318,8 @@ TodoList ComputeActionsToDo(Manifest this_manifest, Manifest next_manifest)
 void CleanUpTempDir(const std::string& temp_dir, const TodoList& todo)
 {
   // This is best-effort cleanup, we ignore most errors.
-  for (const auto& download : todo.to_download)
-    File::Delete(temp_dir + DIR_SEP + HexEncode(download.hash.data(), download.hash.size()));
+  for (const auto& [_filename, hash] : todo.to_download)
```

Comment:
Even though this is unused, please don't use variables that start with an underscore. The C++ standard reserves many of those (with varying rules and scopes) and could break at any given moment.
_I don't think this particular one is considered reserved, but it feels out-of-place to me._

Reply:
How should I mark unused variables? There are very many when using structured bindings.

Reply:
I'd just name them as you'd name a used one, plus potentially apply `[[maybe_unused]]` if it raises an unused variable warning (see the other remark.)

Reply:
There are no warnings raised, I just want it to be clear which parts are actually in use.